lib/fuse_lowlevel.c

```diff
@@ -1955,10 +1969,10 @@ static void do_init(fuse_req_t req, fuse_ino_t nodeid, const void *inarg)
 			bufsize);
 		bufsize = FUSE_MIN_READ_BUFFER;
 	}
+	se->bufsize = bufsize;
```

Comment:
If `bufsize` is only modified in the new `if` statement above, then it would make more sense to move the assignment there as well, IMO.

Reply:
No, bufsize is also modified two lines above this statement, in case it is lower than FUSE_MIN_READ_BUFFER.